### ISSUE TITLE
Mangadex setting for ignoring multiple releases per chapter

### ISSUE THREAD

**Issue by vodkapmp** [metafates/mangal#162]
### Feature Description

Mangadex often has multiple releases from different groups for the same chapters, and mangal will then download all of them, I get that this could sometimes be something you want, but when there's 2-3 different groups for multiple chapters in a row it gets a bit tiresome when you're reading it.

### Solution you would like

A setting for mangadex to only grab one release per chapter.
Preferably configurable between oldest/newest release. And if possible maybe a "same group as previous grabbed chapter"

**Comment by eswiig** [comment]
Yes PLEASE! this is one of the reasons I have started to avoid using MangaDex, which used to be my favorite provider. Other than the rate limiting issue.

**Comment by shinji257** [comment]
I'd like to add a related suggestion but if you would rather this be a separate suggestion then I can make a ticket instead.
Expand chapter name templates to include group so that when downloading multiple sources of the same chapter it will let you know what group submitted that one.  This is especially important when multiple groups did actually agree to the same chapter name or just didn't put in a name in the first place.

Examples: 
Villain - https://mangadex.org/title/7b4cc18a-fc58-4818-ac6a-712147be94c0
Spider-Man: Octopus Girl - https://mangadex.org/title/77e12106-cd14-41eb-8fcf-eb3ba7a69d92
